### PR TITLE
Show natural opg journeys

### PR DESCRIPTION
**Pull request by azime**
Jira: 
[NAV-2761](https://navitia.atlassian.net/browse/NAV-2761)
[NAV-2762](https://navitia.atlassian.net/browse/NAV-2762)
[NAV-2763](https://navitia.atlassian.net/browse/NAV-2763)

[NAV-2761]: https://navitia.atlassian.net/browse/NAV-2761?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[NAV-2762]: https://navitia.atlassian.net/browse/NAV-2762?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[NAV-2763]: https://navitia.atlassian.net/browse/NAV-2763?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: source/jormungandr/jormungandr/olympic_site_params_manager.py [hunk 1]
```diff
@@ -56,6 +56,10 @@ def __repr__(self):
         return "{}-{}-{}".format(self.instance_name, self.s3_object.key, self.s3_object.e_tag)
 
 
+def has_applicable_scenario(api_request):
+    return True if api_request.get("olympic_site_params") else False
```
Comment (xlqian) [inline source/jormungandr/jormungandr/olympic_site_params_manager.py]:
```suggestion
    return bool(api_request.get("olympic_site_params"))
```

File: source/jormungandr/jormungandr/olympic_site_params_manager.py [hunk 2]
```diff
@@ -103,6 +107,18 @@ def get_dict_scenario(self, poi_uri, key, datetime):
             return self.build_olympic_site_params(scenario_name, data)
         return {}
 
+    def get_show_natural_opg_journeys(self, conf_additional_parameters, query_show_natural_opg_journeys):
+        return (
+            query_show_natural_opg_journeys
+            if query_show_natural_opg_journeys is not None
+            else conf_additional_parameters.get("show_natural_opg_journeys", False)
+        )
```
Comment (xlqian) [inline source/jormungandr/jormungandr/olympic_site_params_manager.py]:
```suggestion
        return query_show_natural_opg_journeys or conf_additional_parameters.get("show_natural_opg_journeys", False)
```

**Review by xlqian — APPROVED** [review]

**Review by pbench — APPROVED** [review]

File: source/jormungandr/jormungandr/olympic_site_params_manager.py (hunk 2 above)
Comment (azime) [reply source/jormungandr/jormungandr/olympic_site_params_manager.py]: Done

File: source/jormungandr/jormungandr/olympic_site_params_manager.py (hunk 1 above)
Comment (azime) [reply source/jormungandr/jormungandr/olympic_site_params_manager.py]: Done

**Pull request merged**